app/controllers/users/phones_controller.rb

```diff
@@ -4,6 +4,7 @@ class PhonesController < ReauthnRequiredController
 
     before_action :confirm_two_factor_authenticated
     before_action :redirect_if_phone_vendor_outage
+    before_action :check_max_phone_numbers_per_account, only: %i[add]
```

Comment:
```suggestion
    before_action :check_max_phone_numbers_per_account, only: %i[add create]
```

I think we need to check for max phone numbers on both routes?